driver/src/main/java/eu/cloudnetservice/cloudnet/driver/permission/DefaultPermissionManagement.java

```diff
@@ -28,18 +28,41 @@
 import lombok.NonNull;
 import org.jetbrains.annotations.Nullable;
 
+/**
+ * This abstract default permission management represents the implementation for the permission management. Methods
+ * doing the calculation for permissions and their result are already implemented and there is no need to implement
+ * those yourself.
+ *
+ * @see eu.cloudnetservice.cloudnet.driver.permission.PermissionManagement
+ * @since 4.0
+ */
 public abstract class DefaultPermissionManagement implements PermissionManagement {
 
+  /**
+   * Gets the child permission management. The default implementation does not allow a child permission management,
+   * therefore it's always null.
+   *
+   * @return the child management, by default null.
+   */
   @Override
   public PermissionManagement childPermissionManagement() {
     return null;
   }
 
+  /**
+   * Gets if the permission management can be overwritten. The default implementation of the permission management is
```

Comment:
Nit: I think renaming the method (and then the word in the docs) from overwritten to overridden might be better 🤔